GVFS/GVFS/CommandLine/MountVerb.cs

```diff
@@ -117,42 +117,45 @@ protected override void Execute(GVFSEnlistment enlistment)
 
                 if (!GVFSPlatform.Instance.KernelDriver.IsReady(tracer, enlistment.EnlistmentRoot, out errorMessage))
                 {
-                    if (GVFSPlatform.Instance.UnderConstruction.SupportsGVFSService)
+                    if (!GVFSPlatform.Instance.KernelDriver.TryLoad(tracer, enlistment.EnlistmentRoot, this.Output, out errorMessage))

```

Comment:
We should always be calling `TryLoad` (at least on Windows), but with the latest changes we'll only call `TryLoad` if `IsReady` return false.

May we should do something like rename `IsReady` to `IsInstalled` and always call `TryLoad`? (or just always call `TryLoad`)

Reply:
Hmmm.  In looking at the code before:
```
IsServiceRunning(tracer) &&
IsNativeLibInstalled(tracer, new PhysicalFileSystem()) &&
TryAttach(enlistmentRoot, out error);
```
It would only run the `TryAttach` on windows if the other 2 would be `true` right?  So on windows it would only run the `TryLoad` if `IsReady` is true?  

Renaming to `IsInstalled` wouldn't make sense on Mac because it is checking if it is loaded and running.  Also on Mac if we call it everytime the user will get the sudo prompt every mount even if the kext is loaded which is not what we want.

I think switching this back to just having the `IsReady` and pass the `TextWriter` to that method and each platform can do the work needed is the best option here.

Reply:
Your analysis looks correct to me, and switching back to IsReady sounds good to me as well.